tests/governance.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+import os
+import getpass
+import time
+import logging
+import multiprocessing
+import shutil
+import subprocess
+from random import seed
+import infra.ccf
+import infra.proc
+import infra.jsonrpc
+import infra.notification
+import infra.net
+import e2e_args
+
+from loguru import logger as LOG
+
+
+def wait_for_node_commit_sync(nodes):
```

Comment:
This is already defined three times in `recovery.py`, `e2e_logging.py` and `e2e_scenarios.py`. Should we move it to `Network` in `ccf.py`?

Reply:
Yes!